docs/software/printing.md

```diff
@@ -60,4 +61,24 @@ Website: <https://octoprint.org>
 Official documentation: <https://docs.octoprint.org>  
 Forum: <https://community.octoprint.org>
 
+## CUPS
+
+CUPS provides a web interface for managing your local or network printers.
+
+![CUPS web interface screenshot](../assets/images/dietpi-software-printserver-cups.png){: width="500" height="385" loading="lazy"}
+
+=== "Access the web interface"
```

Comment:
Harmonization: In many other documents we call it "Access to the web interface". Would you please also change this in the octoprint section?

Reply:
Yes